container-guide/concepts/containers-verify.xml

```diff
@@ -48,7 +48,7 @@
     <para>
       To verify a &bcia;, run Cosign in the container. The command below
       fetches the signing key from the &suse; server and uses it to verify the
-      latest &bcia;-Base container image.
+      latest &bcia;-Base container image. 
```

Comment:
The output is _[...] verify the latest SLE BCI-Base container image._ 
Maybe use the &sle; &bci; combination here instead?